cmd/slave/main.go

```diff
@@ -1,17 +1,20 @@
 package main
 
 import (
-	"flag"
 	"log"
 
+	flag "github.com/ogier/pflag"
+
 	"github.com/aaronang/cong-the-ripper/lib/slave"
 )
 
 func main() {
 	log.Println("slave starting...")
 	portPtr := flag.String("port", "8080", "Web server port")
+	masterIpPtr := flag.String("master-ip", "localhost", "Ip address of the master")
+	masterPortPtr := flag.String("master-port", "8080", "Port of the master")
```

Comment:
I though Kelong removed the `Ptr` part. No need to indicate that it is a pointer.